src/EnergyPlus/Psychrometrics.hh

```diff
@@ -719,8 +719,8 @@ namespace Psychrometrics {
 		// FUNCTION LOCAL VARIABLE DECLARATIONS:
 
 		Int64 const Tdb_tag( bit::bit_shift( TRANSFER( T, Grid_Shift ), -Grid_Shift ) ); // Note that 2nd arg to TRANSFER is not used: Only type matters
-//		Int64 const hash( bit::bit_and( Tdb_tag, psatcache_mask ) ); //Tuned Replaced by below
```

Comment:
OK, well it was changed during [this commit](https://github.com/NREL/EnergyPlus/commit/7dff6199beba6bcf7eff4ee50ef0b27be9f159df).  I'll revert the change and pull out bit.hh from the unit test file below and see what happens.  Other than these issues, this branch does look really good though.  Thanks for the efforts in getting unit tests included and also objectifying the code.